src/bittybuzz/src/BittyBuzzMessageHandler.cpp

```diff
@@ -180,6 +181,19 @@ bool BittyBuzzMessageHandler::handleGenericResponse(const GenericResponseDTO& re
     return true;
 }
 
+bool BittyBuzzMessageHandler::handleBuzzMessage(const BuzzMessageDTO& msg) {
+    (void)msg;
+    bbzmsg_payload_t payload;
+    payload.buffer = const_cast<uint8_t*>(msg.getPayload().data());
```

Comment:
If we don't copy the message content, is there a risk the pointer value changes before the BBVM has time to ingest the message?

Reply:
The bbzinmsg_queue_append deserializes it, so it copies the data and doesn't just keep a pointer. If we would copy it in a bbzmsg_paylaod_t it would be a useless copy, so we save a couple cpu cycles with the const cast